### ISSUE TITLE
[Feature Request] Implement `show()` for Value

### ISSUE THREAD

**Issue by nemo83** [Hyperion-BT/helios#65]
Low priority,

as a workaround
```javascript
  func print_value(value: Value, valueName: String) -> () {
    print(valueName + ", lovelace: " + value.get_lovelace().show());
    // Map[MintingPolicyHash]Map[ByteArray]Int
    value
      .get_assets()
      .to_map()
      .for_each( (mph: MintingPolicyHash, tokens: Map[ByteArray]Int) -> {
        print(valueName + ", mph: " + mph.show());
        tokens
          .for_each((tokenName: ByteArray, amount: Int) -> {
            print(valueName + ", tokenName: " + tokenName.decode_utf8() + ", amount: " + amount.show())
          })
      })
  }
```
and how to call it:
```javascript
print_value(my_value, "something to explain what this value is");
```

Log examples:
```shell
input.value, lovelace: 2000000
input.value, mph: 7dd80d5b5d7d94b56eac097e271a710ca9ebbe28ba9a7d98018b7df4
input.value, tokenName: My Cool NFT, amount: 1
```

**Comment by christianschmitz** [comment]
Implemented in v0.13.0 of the library: https://www.hyperion-bt.org/helios-book/lang/automatic-methods.html#copy

**Comment by nemo83** [comment]
Works like a charm. 

Closing
